test/lit/passes/O_gc.wast

```diff
@@ -0,0 +1,43 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+;; RUN: wasm-opt %s -O --all-features -S -o - | filecheck %s
+;; RUN: wasm-opt %s -O --all-features --nominal -S -o - | filecheck %s
+;; RUN: wasm-opt %s -O --all-features --ignore-implicit-traps -S -o - | filecheck %s --check-prefix=TRAPS
+;; RUN: wasm-opt %s -O --all-features --ignore-implicit-traps --nominal -S -o - | filecheck %s --check-prefix=TRAPS
```

Comment:
How about IIT or IGNORE-TRAPS instead of TRAPS? TRAPS feels a little odd as it will *not* trap / assumes there are no traps. But I don't feel strongly, lgtm either way.

Reply:
I was trying to match the number of characters in CHECK so that every lined up, but I agree that the language could be more clear. Will fix.

Reply:
Oh, I see... how about `ITRAP`?

Reply:
Looking at the new output, I actually like that the two output streams are differentiated by different indentation, so I'm happy the way it is.